assets/js/main.js

```diff
@@ -290,6 +290,32 @@ function elasticSearch(query, callback) {
 
 var searchSelectionId = 0;
 var searchListingLength = 0;
+function isNumeric(n) {
+    return !isNaN(parseFloat(n)) && isFinite(n);
+}
+function categoriseResult(post) {
+    if (!post.url_path || !post.url_path.raw) {
+        return ['home'];
+    }
+    var dir1 = post.url_path.raw.split('/')[1];
+    if (isNumeric(dir1)) {
```

Comment:
Could we use a regex instead to check if it's a string of digits? Since `dir1` shouldn't be a float at all.